.github/workflows/next-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release from next branch
```

Comment:
Can we be more explicit.
Next release and release are a bit confusing I find.
Can we agree that: 
- a `release` is from the `main` branch.
- a `release-candidate` is from the `release-candidate' branch for example? 

in both case they would be a concept of next release. That's  I think it would be less confusing that way?

Reply:
@cquemin You want to have a `release-candidate` branch instead of `next` branch for the release candidate releases?  I'm not fussed about the name, but we should be consistent across all the repos